### ISSUE TITLE
Catalog API - organization_list 400

### ISSUE THREAD

**Issue by rossjones** [GSA/datagov-wptheme#294]
When I make a request as follows:

```
curl http://catalog.data.gov/api/3/action/group_list 
```

it succeeds.  but when I make the following request:

```
curl http://catalog.data.gov/api/3/action/organization_list
```

it fails (and it shouldn't) and complains that there is no request body.  Changing my request to:

```
curl -X POST -d '{}' http://catalog.data.gov/api/3/action/organization_list
```

tells me

```
Bad request - Action name not known: organization_list
```

but it is documented at the link from data.gov.uk/developers/api and the corresponding organization_show exists (and works)

```
curl http://catalog.data.gov/api/action/organization_show -d '{"id": "fha-dot-gov"}'
```

**Comment by philipashlock** [comment]
Also wondering if things like `organization_show` should work with GET rather than only POST. It seems to work with current CKAN version.

**Comment by dolaramb** [comment]
Hasn't this issue been fixed yet?
http://catalog.data.gov/api/3/action/organization_list still gives a HTTP 400 error.

**Comment by kvuppala** [comment]
@dolaramb @rossjones  issue might be related to the specific ckan core version data.gov is using, we are checking in detail to see why it breaks, will provide an update with more info. 

In the meantime the below data can be used as well for the current listing of federal agencies.
https://idm.data.gov/fed_agency.json

**Comment by kvuppala** [comment]
Since we are considering the CKAN 2.2.1 upgrade for catalog, it will be ideal to pick this up in that release, for now moving to 2.14

**Comment by rossjones** [comment]
Just in case you weren't aware, CKAN 2.3 is due in the next few weeks, and if that is too new, there are likely to be patches to 2.2.x

**Comment by hkdctol** [comment]
@kvuppala can this be closed based on CKAN upgrade to 2.3?

**Comment by kvuppala** [comment]
@hkdctol Yes, can be closed 

The org list query http://catalog.data.gov/api/3/action/organization_list works fine now.
